.circleci/config.yml

```diff
@@ -1290,6 +1305,43 @@ jobs:
           root: '.'
           paths: [ './extensions' ]
 
+  compile_extension_buster:
```

Comment:
Why is it not possible to use the same `centos-6` images? Is it centos 6 too old to build PHP 8.1? We use it to build 8.0.

The reason why we are doing the build on a separate image, lighter image is that we had a security audit of the way we install and versions we install. It should be preferable to stick to those images.

If not possible, we can build equivalent centos:7 (or buster) images and ask for auditing.

Since we are actually shipping it, I suggest to do it before we merge, or have a separate conversation on why it is safe to proceed like this.

Reply:
Yes, it's too old due to openssl. CentOS 7 also is too old. We'd need at least CentOS 8. The alternative obviously would be compiling openssl manually.

Reply:
We compile quite a few things in the base image https://github.com/DataDog/dd-trace-php/blob/master/dockerfiles/ci/centos/6/Dockerfile#L38-L70

Can we use a similar approach on the https://github.com/DataDog/dd-trace-php/blob/master/dockerfiles/ci/centos/6/Dockerfile_php8 file only, instead of doing it in the base image?

Reply:
FYI, compiling OpenSSL manually for static compilation is not hard -- in fact I do it for the profiler.